src/app/main.cpp

```diff
@@ -253,6 +253,11 @@ int main(int argc, char *argv[])
         app->setAttribute(Qt::AA_DontShowIconsInMenus);
 #endif
 
+#ifndef Q_OS_MACOS
```

Comment:
There is `#if defined(Q_OS_MACOS)` just above it. Why not use `#else` here?

Reply:
Of course, you're right. Fixed.